package.json

```diff
@@ -1,26 +1,34 @@
 {
   "name": "lavadome",
   "version": "0.0.0",
+  "private": "true",
   "description": "Secure DOM trees isolation and encapsulation leveraging ShadowDOM",
   "main": "packages/index.mjs",
+  "workspaces": [
+    "packages/*",
+    "demo"
+  ],
   "devDependencies": {
-    "@wdio/cli": "^7.5.3",
-    "@wdio/local-runner": "^7.5.5",
-    "@wdio/mocha-framework": "^7.5.3",
-    "@wdio/spec-reporter": "^7.5.3",
-    "chromedriver": "^119.0.1",
-    "geckodriver": "^3.2.0",
-    "wdio-chromedriver-service": "7.0.0",
-    "wdio-safaridriver-service": "^2.0.0",
-    "wdio-geckodriver-service": "^4.0.0"
+    "@wdio/cli": "^8.27.0",
+    "@wdio/local-runner": "^8.27.0",
+    "@wdio/mocha-framework": "^8.27.0",
+    "@wdio/spec-reporter": "^8.27.0",
+    "chromedriver": "^120.0.0",
+    "geckodriver": "^4.3.0",
+    "wdio-chromedriver-service": "8.1.1",
+    "wdio-safaridriver-service": "^2.1.1",
+    "wdio-geckodriver-service": "^5.0.2",
```

Comment:
I bumped the deps to their latest versions, but there are some major version updates here. Anything that might cause an issue?